api/urls.py

```diff
@@ -45,5 +46,7 @@
     path('add-friend-to-trip/<int:trip_id>/<int:user_id>', trip_views.add_friend_to_trip, name="add-friend-to-trip"),
 
     # Feedback
-    path('add-feedback', feedback_views.add_feedback, name="add-feedback"),
+    path('add-feedback',  feedback_views.add_feedback, name="add-feedback"),
+    path('user-feedback', feedback_views.get_feedback_all, name="user-feedback"),
```

Comment:
Use `get-all-user-feedback`
Also have same name and method name inside views class to avoid confusion